float32 newLat
float32 newLong
